cmd/adapters/ingresses/ingresses.go

```diff
@@ -107,6 +116,32 @@ func (c *VampCloudAnansiIngressClient) PostIngress(ingress models.Ingress) (int6
 
 }
 
+func checkPreviewRoute(routes []*dto.Route) error {
+
+	re := regexp.MustCompile(`%%(.*)%%`)
+	for _, v := range routes {
+		path := strings.Split(v.Path, "/")
+
+		for _, v2 := range path {
+			result := re.FindSubmatch([]byte(v2))
+			if len(result) > 0 && !validPlaceholder(result[1]) {
```

Comment:
* shall we check length against 1 instead, e.g. `len(result) > 1` just to make sure that result[1] is always safe?
* why not to use `FindStringSubmatch ` instead? (to skip type casting)